internal/server/handler_v1.go

```diff
@@ -183,38 +183,78 @@ func (s *Server) VariableInfo(
 	return info.VariableInfo(ctx, in, s.store)
 }
 
+var storeBulkVariableInfo = info.BulkVariableInfo
+
+var remoteBulkVariableInfoResponse = func(
```

Comment:
the name should be bulkVariableInfoRemoteFunc?

Also given the size of this file and easier locating these stub functions, maybe have a new file like handler_v1_func.go to hold these function variables.

Reply:
Done.